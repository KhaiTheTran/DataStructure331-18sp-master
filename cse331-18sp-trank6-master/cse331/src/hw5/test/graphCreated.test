# Tests initial graph.

# Create a graph
CreateGraph Creategraph

# Print the nodes in the graph
ListNodes Creategraph
