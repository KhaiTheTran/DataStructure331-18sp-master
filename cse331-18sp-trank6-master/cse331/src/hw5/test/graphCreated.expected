# Tests initial graph.

# Create a graph
created graph Creategraph

# Print the nodes in the graph
Creategraph contains:
